src/controllers/seller controller/sellerController.ts

```diff
@@ -53,52 +149,53 @@ class SellerController {
       res.status(401).json({ message: 'Unauthorized: Token is missing' });
       return;
     }
-
+  
     try {
       const decodedToken = jwt.verify(token, process.env.VERIFICATION_JWT_SECRET as string) as JwtPayload;
       const userId = decodedToken.userId;
```

Comment:
Same here. If there is a middleware checking if a user is a seller it would be easier and reusable.